provider/k8s/template/app/service.yml.tmpl

```diff
@@ -85,9 +85,6 @@ spec:
           - "svc.cluster.local"
           - "cluster.local"
       {{ end }}
-      {{ if .Service.Agent.Enabled }}
```

Comment:
Did you test agent [communication](https://docs.convox.com/configuration/agents/) after this change?
If it doesn't work we need to provide another solution. Probably having a k8s service for the agent like we do for other apps(if it doesn't create one already)

Reply:
hey @heronrs, I'm still making manual testing this. I created a draft PR to have a look at the checks down here. 
On the first tests, there was no change in the communication between pods in the same rack, but I'm not confident 100%.

Reply:
Hey,
> It worked, another processes running in another instances are not visible anymore.

What do you mean by that? Were such processes visible before the change?
I'm not sure about the svc because routing will go round robin way and by using the instance IP you guarantee the service will always reach the agent running in the same node.

Reply:
there was one case that I could communicate: 

-  I had an deployed app, I applied the change to remove the hostNetwork from the agent, and I still could communicate. Even I could make a request for an agent running at the same node.
-  Now, I deployed another app with this change already applied, this way I could not communicate by IP with another nodes in the same rack.

Reply:
I think being able to communicate with the same node is fine - we should document that in the release notes

As for the svc is not much about security but the fact that workloads on the same node could be reaching an agent in a different node - you could have agents in one node receiving a lot of traffic while others could be idle.

Reply:
got it. I'll update the docs as well and include here, on this PR.